### ISSUE TITLE
Venv conflict

### ISSUE THREAD

**Issue by Systemac** [piyushK52/comfy_runner#19]
Hello !

First, thank's for you work !

Would it be possible to use the venv from an existing comfyui installation?

**Comment by piyushK52** [comment]
Yeah you can install the comfy_runner requirements in your comfyui venv
